### PR TITLE
SecureDrop 1.1.0~rc1 changelog

### PR DESCRIPTION
**Pull request by emkll**
## Status

Ready for review

## Description of Changes

Towards #4887 , updates the changelog for 1.1.0-rc1

## Testing

- [ ] The changelog is accurate

### If you made changes to the system configuration:

- [x] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -2,6 +2,30 @@
...
 
+### Web Applications
+
+* Reordered and updated wording of manage.py - #4850, #4858
+* Remove python 2 support in securedrop server code - #4859
+
+### Operations
+
+* Improve specification of securedrop-app-code dependencies - #4876
+* Added check for admin password in GUI updater - #4877
```
Comment (redshiftzero) [inline changelog.md]: should be in admin tails workstation section?

File: changelog.md [hunk 2]
```diff
@@ -2,6 +2,30 @@
...
+* Updated dependencies for app server and admin workstation - #4865, 4884
+* Fixes to packaging #4870, #4871
```
Comment (redshiftzero) [inline changelog.md]: Nit: formatting of the changelog lines is a bit different than the rest of the changelog, i.e. in the past it's been:

`* Description of item (#NNNN).`

File: changelog.md (hunk 1 above)
Comment (emkll) [reply changelog.md]: good catch, fixed

File: changelog.md (hunk 2 above)
Comment (emkll) [reply changelog.md]: I mistakenly copied the formatting of earlier releases, now fixed

**Pull request merged**